### PR TITLE
larger genome alignment should go to basement for now

### PR DESCRIPTION
**Pull request by yumisims**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/treeval/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by DLBPointon** [comment]
LGTM
I'll approve once it passes tests

**Review by weaglesBio — APPROVED** [review]

**Comment by yumisims** [comment]
I knew it was something small!!

File: conf/base.config [hunk 1]
```diff
@@ -128,15 +128,16 @@ process {
...
         time   = { check_max( 10.h     * task.attempt, 'time'    ) }
     }
 
     // Standard parameters, covers most insecta
     withName: '.*:.*:READ_COVERAGE:MINIMAP2_ALIGN' {
         cpus    = { check_max( 20       * 1, 'cpus'                 ) }
         memory  = { check_max( 50.GB    * task.attempt, 'memory'    ) }
-        time    = { check_max( 20.h     * task.attempt, 'time'      ) }
+        time    = { check_max( 1.h      * (reference.size() < 1e9 ? 10 : reference.size() < 2.5e9 ? 30 : 300), 'time'      ) }
+        //time   = { check_max( 30.h    * task.attempt, 'time'    ) }
```
Comment (DLBPointon) [inline conf/base.config:140]: Remove the unneeded time line

**Review by DLBPointon — CHANGES_REQUESTED** [review]
Let's wait until the tests pass and then we can just commit this change and merge straight away

**Pull request merged**